src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,50 +1,148 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.NoSuchElementException;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private T value;
+    private MyLinkedList<T> next;
+    private MyLinkedList<T> prev;
```

Comment:
let's use Nodes in our linkedlist as in original implementation
create inner class Node with these three fields
https://habr.com/ru/post/337558/